### DISCUSSION TITLE
Custom Compute Budget in Phantom wallet

### DISCUSSION THREAD

**Discussion by vakkuda** [phantom/docs#212] [General]
Please add a custom Compute budget to so that way we can up a fee to have less failed transactions. It would be like custom Gwei settings in Metamask by adding a custom compute budget with the transfer instructions for custom lamport and units.

**Comment by adamdelphantom** [comment]
Hello @vakkuda thank you for writing in!

Please consider creating a feature request [here](https://phantom.canny.io). This way others can vote to add their support to your request.
